lib/services/lwm2mHandlers/registration.js

```diff
@@ -57,7 +57,7 @@ function mapConfig(device, configuration, callback) {
             service: configuration.service,
             subservice: configuration.subservice,
             lazy: configuration.lazy,
-            attributes: configuration.attributes,
```

Comment:
Related with https://github.com/telefonicaid/iotagent-node-lib/issues/588... 

I understand this is an internal variable not related with the keyword used in payload (i.e. `"attributes`"). Is my interpretation correct?

Reply:
In fact, this is the field used in the payload to describe attributes. So this was an error caused by https://github.com/telefonicaid/iotagent-node-lib/issues/588

Reply:
> In fact, this is the field used in the payload to describe attributes

You mean to `configuration.attributes`, right?

Reply:
Yes, that's right

Reply:
NTC